src/main/features/claim/paths.ts

```diff
@@ -39,6 +39,7 @@ export class Paths {
   static readonly eligibilityClaimantAddressPage = new RoutablePath('/claim/eligibility/claimant-address')
   static readonly eligibilityDefendantAddressPage = new RoutablePath('/claim/eligibility/defendant-address')
   static readonly eligibilityOver18Page = new RoutablePath('/claim/eligibility/over-18')
+  static readonly eligibilityDefendantAgeOptionPage = new RoutablePath('/claim/eligibility/over-18-defendant')
```

Comment:
Please rename value to end with `/defendant-age` and property name to get rid of `Option`, it should be `eligibilityDefendantAgePage `